src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -277,9 +277,17 @@ bool Replayer<I>::get_replay_status(std::string* description,
       matching_remote_snap_it !=
         m_state_builder->remote_image_ctx->snap_info.end()) {
     root_obj["syncing_snapshot_timestamp"] = remote_snap_info->timestamp.sec();
-    root_obj["syncing_percent"] = static_cast<uint64_t>(
-        100 * m_local_mirror_snap_ns.last_copied_object_number /
-        static_cast<float>(std::max<uint64_t>(1U, m_local_object_count)));
+
+    if (m_local_object_count > 0) {
+      root_obj["syncing_percent"] =
+	100 * m_local_mirror_snap_ns.last_copied_object_number /
+	m_local_object_count;
```

Comment:
indentation
```suggestion
	  100 * m_local_mirror_snap_ns.last_copied_object_number /
	  m_local_object_count;
```

Reply:
I'm not sure I understand. This matches the indentation in other blocks.

Reply:
I'm trying to get librbd to a place where at least new contributions are consistent in style and can be formatted automatically (according to the Google C++ style guide, which Ceph tries to adhere to in most places) with as few exceptions as possible.  The continuation indent width is 4 spaces.